components/cli/internal/test/bal_executor.go

```diff
@@ -109,7 +109,7 @@ func (balExecutor *MockBalExecutor) Test(fileName string, args []string, envVars
 }
 
 // Build mocks execution of ballerina run for cellery init on an executable bal file.
-func (balExecutor *MockBalExecutor) Init(projectName string) error {
+func (balExecutor *MockBalExecutor) Init(workingDir string, projectName string, moduleName string) error {
```

Comment:
Method parameters can be rewritten as "workingDir, projectName, moduleName string"

Reply:
Fixed in 8f88566